src/joint.dia.cell.js

```diff
@@ -927,7 +935,7 @@ joint.dia.CellView = joint.mvc.View.extend({
         if (this._collection) {
             // we don't want to trigger event on model as model doesn't
             // need to be member of collection anymore (remove)
-            this._collection.trigger('batch:stop', { batchName: 'pointer' });
+            this._collection.stopBatch('pointer');
```

Comment:
GraphCells have stopBatch, should we move the code managing the Batches to the Graph instead?